Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM hayd/alpine-deno:1.2.2
+
+RUN apk add --no-cache python py-pip
+RUN pip install awscli
+
+WORKDIR /app
+
+COPY import_map.json .
+COPY deps.ts .
+RUN deno cache --unstable --importmap import_map.json deps.ts
+COPY test_deps.ts .
+RUN deno cache --unstable --importmap import_map.json test_deps.ts
```

Comment:
The import map is only used to work around the "std@v" problem? Can that be removed now that you fixed that regression?

Reply:
Yes, ill do that in a separate PR